src/main/java/com/blackducksoftware/integration/jira/task/issue/handler/DataFormatHelper.java

```diff
@@ -102,17 +105,21 @@ public String getIssueDescription(final IssueCategory issueCategory, final Strin
         return issueDescription.toString();
     }
 
-    public String createVulnerableComponentsLink(final String componentName, final String baseVulnerableComponentsLink) {
-        if (StringUtils.isNotBlank(componentName)) {
-            final String encodedComponentName;
+    public Optional<String> createUrlWithComponentNameQuery(final String url, final String componentName) {
+        if (StringUtils.isNotBlank(componentName) && StringUtils.isNotBlank(componentName)) {
```

Comment:
Why is this if checking the same thing twice?

Reply:
Oops, I refactored the variable order, but somehow screwed this up.